README.md

```diff
@@ -168,6 +168,15 @@ Default:
 
 Glob pattern(s) for finding relevant resources inside `cwd`. If set, the default patterns will be replaced.
 
+##### compatVersion
+Type: `string`  
+Default: `'edge'`
```

Comment:
`edge` (without single quotes, as type `string` should make clear that it is a string).